druid-shell/src/backend/gtk/application.rs

```diff
@@ -84,7 +84,12 @@ impl Application {
     }
 
     pub fn get_locale() -> String {
-        glib::get_language_names()[0].as_str().into()
+        let mut locale: String = glib::get_language_names()[0].as_str().into();
```

Comment:
this index would panic if there `get_language_names()` is empty. Is that possible?

Reply:
`Computes a list of applicable locale names, which can be used to e.g. construct locale-dependent filenames or search paths. The returned list is sorted from most desirable to least desirable and always contains the default locale "C".`

If druid fails to parse the locale it will be "en_US" so this should work in all cases.